firewall.go

```diff
@@ -54,9 +54,10 @@ type FirewallRuleConfig struct {
 
 // FirewallConfig is how you specify the details when creating a new firewall
 type FirewallConfig struct {
-	Name      string `json:"name"`
-	Region    string `json:"region"`
-	NetworkID string `json:"network_id"`
+	Name        string `json:"name"`
+	Region      string `json:"region"`
+	NetworkID   string `json:"network_id"`
+	CreateRules bool   `json:"create_rules,omitempty"`
```

Comment:
Shouldn't be a pointer here too?

Reply:
For reference, a comment would be nice too. 
If `nil` is sent, it implies rules will be created. 
Essentially, unless user explicitly sends false, we'll create rules for them

Reply:
By default the variable will be set to false, won't it?

Reply:
Yes true